dev-python/latexcodec/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <longdescription>
```

Comment:
I usually prefer maintainers first but it looks like this is legal.

Reply:
Put maintainers first